n2kMapper.js

```diff
@@ -44,6 +44,7 @@ var toDelta = function (n2k, state) {
     return result
   } catch (ex) {
     console.error('Unable to convert:' + ex.message + ':' + JSON.stringify(n2k))
+    console.error(ex.stack)
```

Comment:
I actually didn't mean to check this in, but I'm constantly having to go in and modify n2kMapper to print the stack trace. Would you prefer to do it with debug?